### DISCUSSION TITLE
3004.388.7_1-gnuton0_alpha1

### DISCUSSION THREAD

**Discussion by gnuton** [gnuton/asuswrt-merlin.ng#581] [Announcements]
##  GNUton's Asus Merlin changelog  ##

Date: April 24, 2024
Release: 3004.388.7_1-gnuton1

  - ADDED: XT_SET kernel module for TUF-AX3000_V2 and other  SDK 5.04 based  models
  - FIXED: Issues flashing RT-AX82U_V2 image

<hr /><em>This discussion was created from the release <a href='https://github.com/gnuton/asuswrt-merlin.ng/releases/tag/3004.388.7_1-gnuton0_alpha1'>3004.388.7_1-gnuton0_alpha1</a>.</em>

**Comment by shiang-luong** [comment]
I hit an issue where the router crashed and restarted after enabling VPN client (and VPN director).  I reverted back, but wanted to see if anyone else had run into that issue.

**Reply by gnuton** [reply]
@shiang-luong can you please share the model where you found this issue? 
What's the version you have reverted back?

**Reply by shiang-luong** [reply]
Ah so sorry.  
Xt8 on the main node only. Reverted back to 3.0.0.4.388.6_0
Let me know if I can help debug in anyway.

**Reply by shiang-luong** [reply]
Ok after I reverted back, my network just didn't seem stable anymore. So I did a factory reset. After the factory reset, I went ahead and upgraded again to the alpha.  So far so good. 
One thing to note. When configuring my VPN director, all my access to the WAN and the GUI interface died right when the gui stated complete.  Work around was to hard reboot the main router.  Let me know if I should file a bug for this

**Comment by SOBON2O** [comment]
Love this project!!! Seriously, I can't thank you enough. I am going to donate. No XMR address?

**Comment by shiang-luong** [comment]
Ok, never had this issue before, but system rebooting via the link under aimesh seems to mess up my wifi/aimesh. Hard reboot on all nodes seems to resolve it.
Xt8 (main node alpha gnuton fw, satellite node current Asus fw)

**Comment by Frozymelon** [comment]
my LAN Manually Assigned IP around the DHCP and VPN rule are wipe out on lastest version  
TUF-AX3000_3004_388.6_0-gnuton1_tuf_puresqubi
